packages/fx-core/src/plugins/resource/sql/managementClient.ts

```diff
@@ -182,23 +182,23 @@ export class ManagementClient {
   }
 
   async addLocalFirewallRule(): Promise<void> {
-    const response = await axios.get(Constants.echoIpAddress);
-    const localIp: string = response.data;
-    const partials: string[] = localIp.split(".");
+    try {
+      const response = await axios.get(Constants.echoIpAddress);
```

Comment:
maybe not related to the error fix. But one question is why we not use os networkInterfaces instead to get ip address. Is there any limitation?

Reply:
We need to get the external IP address of the machine to config the firewall rule. The network interface in machine provides the internal  IP address which is not public IP address and usually is translated by NAT.